runelite-client/src/main/java/net/runelite/client/plugins/fairyring/FairyRingPlugin.java

```diff
@@ -68,6 +75,7 @@
 )
 public class FairyRingPlugin extends Plugin
 {
+	public static final String CONFIG_GROUP_TAGS = "fairyringtags";
```

Comment:
Instead of using a completely separate config group for tags, this can use an additional qualifier, eg. `FairyRingConfig.GROUP + ".tags"`, or something of the like. You can reference the bank tags plugin to see how it manages its config.